readme.md

```diff
@@ -285,6 +286,12 @@
 - [Information Retrieval](https://github.com/harpribot/awesome-information-retrieval#readme) - Learn to develop your own search engine.
 
 
+## Natural Sciences
+
+- [Bioinformatics](https://github.com/danielecook/Awesome-Bioinformatics#readme)
+- [Materials Informatics](https://github.com/tilde-lab/awesome-materials-informatics#readme)
```

Comment:
> The entry in the Awesome list should:
* Include a short description about the project/theme of the list. **It should not describe the list itself.**